src/pages/authorization/authorization-in-practice/index.mdx

```diff
@@ -36,31 +36,40 @@ To use any method from the Agent Chat, Customer Chat, and Configuration API, you
 
 <img
   alt="LiveChat Developer Console"
-  src="/images/authorization/livechat-authorization-block-2x.png"
-  width="800px"
-  height="458px"
+  src="/images/authorization/livechat-authorization-in-practice-building-block.png"
+  width="1291px"
+  height="910px"
+/>
+
+However, before that, you must decide if you are building a JavaScript or server-side app. We will use the Server-side app in our guide.
```

Comment:
```suggestion
However, before that, you must decide if you are building a JavaScript or server-side app. We will use the server-side app in our guide.
```

Reply:
ajajajaj you are totally right! I do not know how I missed it, as I was sure I clicked all the links in the document! Fixing it now